tests/flow/new_react/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -865,11 +865,7 @@ var ReactClass = React.createClass({
   render: function(): any {
     // Any state access here seems to make state any
     this.state;
-    return (
-      <div>
-        {this.state.bar.qux}
-      </div>
-    );
+    return <div>{this.state.bar.qux}</div>;
```

Comment:
@karl is there a way to prevent these kinds of changes? or are they intentional?